### DISCUSSION TITLE
'pending' status isn't logged as DEBUG

### DISCUSSION THREAD

**Discussion by jdt** [renovatebot/renovate#32388] [Request Help]
### How are you running Renovate?

Self-hosted Renovate

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Azure DevOps, 38.122.0

### Please tell us more about your question or problem

In processBranch, not all return statements have an associated logger.debug line so we can see why a branch would not be made into a PR. For example, ['already-existed'](https://github.com/renovatebot/renovate/blob/0ec39652cd211c719790e0108fa2f60ffe5b701a/lib/workers/repository/update/branch/index.ts#L168) and ['needs-approval'](https://github.com/renovatebot/renovate/blob/0ec39652cd211c719790e0108fa2f60ffe5b701a/lib/workers/repository/update/branch/index.ts#L192) will log, but ['pending'](https://github.com/renovatebot/renovate/blob/0ec39652cd211c719790e0108fa2f60ffe5b701a/lib/workers/repository/update/branch/index.ts#L184) will not. This can be very confusing, especially when updating a new NPM package where the minimumReleaseAge parameter comes into play. There should be a debug logging call to indicate this.

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by jdt** [comment]
I'll gladly send over a PR for this small change but for some reason I can't turn this discussion into an issue

**Reply by viceice** [reply]
you don't need an issue, just send the PR

**Reply by jdt** [reply]
Thanks, see https://github.com/renovatebot/renovate/pull/32390
